Blog post about the war in Ukraine (#267)
mixxxdj/website: content/news/XXXX-XX-XX-russians-war.md

> title: Russian's war on Ukraine

Reviewer: Resist to take sides. It is a war between two nations, which is bad. We should not blame one or the other. The situation requires deescalation.

Reviewer 2: We are not politicians and can't do much in regards to deescalation. IMO the situation is pretty clear on who the aggressor is in this war and talking around that is simply unjust towards the people of the invaded nation.

Reviewer 3: I agree with your other comments to use more neutral wording.
However, we _are_ taking sides with this post, that's what it's all about, isn't it? It's clearly Russia's invasion of Ukraine, Russia's war on Ukraine, thus I don't see why to water down the heading when we mention exacly that in the text below.

Reviewer: Maybe some native speaker could comment if the wording reflects this position and isn't more offensive than needed? Even in German I would neither be competent nor familiar with describing war situations and the roles of the parties involved.

Reviewer: I guess calling it an "invasion" is undisputed and more specific, so I would prefer "Russia's invasion of Ukraine".

Author: Done

authors: Daniel Schürmann
tags: russian ukraine war
status: draft
comments: yes

> Since 2022-02-24 we receive horrible news from Ukaine, which have become even worse since then. As an organization committed to freedom we feel the urge speak up and show solidarity for Ukraine.

Reviewer: Don't exaggerate, keep it and simple and neutral: "Since 2022-02-24 we are receiving concerning news from Ukraine." (There was also a typo "Ukaine")

Author: I see news where whole families including childs are murdered or a group of young 150 solders has been killed. That is horrible. Calling this concerning feels misplaced. The situation was already concerning before 2022-02-24 IMHO.

> Russia's brutal invasion of Ukraine cannot be justified by anything and has already been condemned by the UN as a violation of international law. The attack creates endless suffering for men, women and children in Ukraine but also for the many Russian soldiers who are sent into a senseless war that they do not want, do not understand and are not even allowed to call it as such.

Reviewer: I recommend to omit any kind of biased or judgemental wording like "brutal" or "horrible". Accusations in any direction will only divide people. That's not what we want to achieve.

Author: OK, I have removed "brutal"

> Mixxx and the FOSS movements respects users' freedom and stands also for freedom in other regards, including free speech and world wide peaceful collaboration. Let's express this also in our DJ sets by integrating tracks from Ukrainian artist, Russian activists or anti-war tracks in general.

Reviewer: -> typo: "worldwide"? I like the phrase "worldwide peaceful collaboration".

Author: Done

title: Russian's war on Ukraine
authors: Daniel Schürmann
tags: russian ukraine war
status: draft
comments: yes

> Since the 2022-02-24 we receive horrible news from Ukaine, which have become even worse since than. This requires that we all speak up and show solidarity for the Ukraine.

Reviewer 4: than -> then

Reviewer 4: for the Ukraine -> for Ukraine

> Since 2022-02-24 we receive horrible news from Ukraine, which have become even worse since then. As an organization committed to freedom we feel the urge speak up and show solidarity for Ukraine.

Reviewer 5: ```suggestion
On 2022-02-24 we received horrible news from Ukraine, which has become even worse since then. As an organization committed to freedom we feel the urge speak up and show solidarity with Ukraine.
```

> Russia's brutal invasion of Ukraine cannot be justified by anything and has already been condemned by the UN as a violation of international law. The attack creates endless suffering for men, women and children in Ukraine but also for the many young Russian soldiers who are sent by a dictator into a senseless war that they do not want, do not understand and are not even allowed to call it such.

Reviewer 4: call it such -> call it as such

> Russia's invasion of Ukraine cannot be justified by anything and has already been condemned by the UN as a violation of international law. The attack creates endless suffering for men, women and children in Ukraine but also for the many Russian soldiers who are sent into a senseless war that they do not want, do not understand and are not even allowed to call it as such.

Reviewer 5: ```suggestion
Russia's invasion of Ukraine is unjustified and has already been condemned by the UN as a violation of international law. The attack creates endless suffering not only for the men, women and children of Ukraine but also for the many Russian soldiers who are sent into a senseless war that they do not want, do not understand and are not even allowed to call as such.
```

> Mixxx and the FOSS movements respects users' freedom and stands also for freedom in other regards, including free speech and worldwide peaceful collaboration. Let's express this also in our DJ sets by integrating tracks from Ukrainian artist, Russian activists or anti-war tracks in general.

Reviewer 5: ```suggestion
Mixxx and the FOSS movement respect users' freedom and also stand for freedom in other regards, including free speech and worldwide peaceful collaboration. As an expression of solidarity, we suggest integrating tracks in our DJ sets from Ukrainian artists, Russian activists or by playing anti-war tracks in general.
```

Reviewer 5: I think we should say a little more here, like:  "Although we are powerless to stop this war, and we are under no illusion that music alone will bring an end to the suffering, we feel it's important to speak out and make our voices heard."

Author: Yes that's a good idea. I have extended the sentence towards supporting all efforts helping Ukrainians.

> Mixxx and the FOSS movements respects users' freedom and stands also for feedome in other regards including free speech. This should we also express in our DJ set's by integrating traks from Ukrain artist, Russian activists or anti war tracks.

Reviewer 4: freedome -> freedom

Reviewer 4: A comma after "in other regards"?

Reviewer 4: express -> be expressed
set's -> sets
Ukrain artist -> Ukranian artists

1. [Бетон - Kyiv Calling](https://www.youtube.com/watch?v=WLCiA38No3Y)
2. [Океан Ельзи - Не твоя війна (Not your war)](https://music.youtube.com/watch?v=QFZV_j5fjEM)
3. [PROBASS ∆ HARDI - ДОБРОГО ВЕЧОРА (Where are you from?)](https://music.youtube.com/watch?v=wir4k8BbX_Q)

> In case you have no access to them contact us on [Zulip](https://mixxx.zulipchat.com)

Reviewer 5: ```suggestion
In case you cannot access them, please contact us on [Zulip](https://mixxx.zulipchat.com)
```

title: Russian's war on Ukraine
authors: Daniel Schürmann
tags: russian ukraine war
status: draft
comments: yes

> Since the 2022-02-24 we receive horrible news from Ukaine, which have become even worse since then. This requires that we all speak up and show solidarity for Ukraine.

Reviewer 3: ```suggestion
Since 2022-02-24 we receive horrible news from Ukaine, which have become even worse since then. As an organization commited to freedom we feel the urge speak up and show solidarity for Ukraine.
```

> Russia's brutal invasion of Ukraine cannot be justified by anything and has already been condemned by the UN as a violation of international law. The attack creates endless suffering for men, women and children in Ukraine but also for the many young Russian soldiers who are sent by a dictator into a senseless war that they do not want, do not understand and are not even allowed to call it as such.

Reviewer 3: ```suggestion
Russia's brutal invasion of Ukraine cannot be justified by anything and has already been condemned by the UN as a violation of international law. The attack creates endless suffering for men, women and children in Ukraine but also for the many Russian soldiers who are sent into a senseless war that they do not want, do not understand and are not even allowed to call it as such.
```

> Putin's propaganda, misinformation and punishment of the opposition has secured him some positive support among russian people. The extent of censorship and political persecution is unprecedented in the recent Russian history.

Reviewer 3: I suggest to remove this paragraph and focus on the actual incidents.

Author: OK, I have removed it.

> Mixxx and the FOSS movements respects users' freedom and stands also for feedome in other regards, including free speech and world wide peaceful collaboration. Let's express this also our DJ set's by integrating tracks from Ukrainian artist, Russian activists or anti war tracks.

Reviewer 3: ```suggestion
Mixxx and the FOSS movements respects users' freedom and stands also for freedom in other regards, including free speech and world wide peaceful collaboration. Let's express this also in our DJ sets by integrating tracks from Ukrainian artist, Russian activists or anti-war tracks in general.
```
